templates/vanilla-sonoma.pkr.hcl

```diff
@@ -0,0 +1,120 @@
+packer {
+  required_plugins {
+    tart = {
+      version = ">= 1.2.0"
+      source  = "github.com/cirruslabs/tart"
+    }
+  }
+}
+
+source "tart-cli" "tart" {
+  # Currently macOS 14 (Sonoma) can only be obtained via a link[1] that
+  # requires authentication, so you have to download it yourself.
+  #
+  # [1]: https://download.developer.apple.com/WWDC_2023/macOS_14_Beta_Restore_Images/UniversalMac_14.0_23A5257q_Restore.ipsw
+  from_ipsw    = "UniversalMac_14.0_23A5257q_Restore.ipsw"
```

Comment:
Can use this link without auth 😜

https://updates.cdn-apple.com/2023SummerSeed/fullrestores/032-94355/CBE8CBE1-750D-487E-A393-B90FEF60CEBA/UniversalMac_14.0_23A5257q_Restore.ipsw

Reply:
Cool, updated the link in https://github.com/cirruslabs/macos-image-templates/pull/81/commits/436789e002c47109822a7605638ae4a4168fbc44.